pkgs/development/libraries/a52dec/default.nix

```diff
@@ -13,6 +13,10 @@ stdenv.mkDerivation rec {
     "--enable-shared"
   ];
 
+  NIX_CFLAGS_COMPILE = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
+    "-std=gnu89"
```

Comment:
If I understand the error correctly it stems from the fact that `inline` gets replaced for `__attribute__ ((__always_inline__))`: it replaces a keyword for attribute annotation. I would say it's a bug to lose a keyword.

I think we can do a workaround of setting the cache value to `ac_cv_c_inline=yes`, say via:

```nix
  configureFlags = [
    "--enable-shared"
    "ac_cv_c_inline=yes"
  ];
```

Can you give it a try and check if it's enough? I would also suggest:
1. applying the fix unconditionally for all targets. Otherwise there is a chance for bug to come back in other forms depending in what `inline` functions are hiding in system headers.
2. adding a comment why the change is needed.

Reply:
Indeed. Thanks!